### PR TITLE
Setup branch

### PR DESCRIPTION
**Pull request by johnspeny**
**Update to be added on Main**

1. Added Read me content 
2. core module 
3. commands.txt to help in the instruction of running some commands
4.

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (alvinMemphis) [inline .idea/.gitignore]: Please add all python to be ignored files .and include ignore for ./idea files

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+/shelf/
```
Comment (alvinMemphis) [inline .idea/.gitignore]: change according to previous comment

**Review by alvinMemphis — CHANGES_REQUESTED** [review]

File: .idea/.gitignore (hunk 2 above)
Comment (johnspeny) [reply .idea/.gitignore]: I have added .idea to gitignore file and removed the cached version using the git command.

Comment (johnspeny) [reply .idea/.gitignore]: @alvinMemphis review

**Review by alvinMemphis — APPROVED** [review]

**Pull request merged**